app/templates/components/add-reminder.hbs

```diff
@@ -0,0 +1,9 @@
+<h2>New Reminder</h2>
```

Comment:
This file has nothing to do with the current issue to "see all reminders". It makes sense that down the road we will need to add reminders, but until that feature is requested please remove extra functionality.